_includes/neurips/2019/header.neurips.2019.main.html

```diff
@@ -15,7 +15,7 @@
         <!-- <a href="#nipsProgram">Program</a> -->
         <a href="#nipsSponsors">Sponsors</a>
         <a href="#nips-org">Organizers</a>
-        <!-- <a href="#">Presenters</a> -->
+        <a href="#">Presenters</a>
```

Comment:
The presenter's link should go to a page reference: "/neurips-2019-presenters" 

See prior year's workshop page for reference:
http://www.latinxinai.org/neurips-2018-presenters

This page still needs to be created at /_pages/neurips_2019/neurips-2019-presenters.html

Use prior year's page as a template found at /_pages/neurips_2018/neurips-2018-presenters.html